templates/web/camden/report/new/form_after_heading.html

```diff
@@ -0,0 +1 @@
+<p>Do not use this to <a href="/faq#emergencies">report emergencies outside of working hours</a>.</p>
```

Comment:
We don't want this template, we have an admin way of doing a reporting page emergency message for new cobrands.